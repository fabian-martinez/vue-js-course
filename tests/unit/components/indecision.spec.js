import Indecision from "@/components/Indecision.vue";
import { shallowMount } from "@vue/test-utils";

describe('', () => {

    let wrapper

    beforeEach(() => {
        wrapper = shallowMount(Indecision)
    })

    test('debe hacer match con el Snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot()
    });

    test('no debe disparar nada', () => {
        console.log('No debe disparar nada');
    });

    test('escribir el "?" debe fisparar el fetch', () => {
        console.log("disparar fetch");
    });

    test('prueba el getAnswer', () => {
        console.log('Prueba el getAnswer');
    });

    test('pruebas de getAnswer falle', () => {
        console.log('prueba que falla el answer');
    });
})