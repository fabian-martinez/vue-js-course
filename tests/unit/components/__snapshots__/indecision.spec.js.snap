// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`debe hacer match con el Snapshot 1`] = `
<!--v-if-->
<div class="bg-dark"></div>
<div class="indecision-container"><input type="text" placeholder="Hazme una pregunta">
  <p>Recuerda terminar con signo de interogaci√≥n (?)</p>
  <!--v-if-->
</div>
`;
